supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+[[red-hat-directory-server-conventions]]
+
+For documentation questions, contact rhel-docs@redhat.com.
+
+[discrete]
+[[access-control-instruction]]
+==== image:images/yes.png[yes] access control instruction (noun)
+*Description*: In an LDAP directory, an access control instruction (ACI) grants or denies access to entries or attributes. Use access control instruction on the first usage and then the abbreviation ACI subsequently.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[attribute]]
+==== image:images/yes.png[yes] attribute (noun)
+*Description*: Each entry in an LDAP directory contains attributes. Object classes in an entry control which attributes in an entry are optional and which are required.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[base-dn]]
+==== image:images/yes.png[yes] base DN (noun)
+*Description*: In an LDAP directory, the base distinguished name (DN) defines the starting point for operations, such as searches.
```

Comment:
By chance is there any guidance on when to write out "base distinguished name (DN)" and when they can just say "base DN"? Because the entry is "base DN" but the description writes it out fully.

Reply:
This gets explained in the DN entry below - "distinguished name" on first usage, then "DN" subsequently.